src/main/java/in/hp/java/userservice/model/User.java

```diff
@@ -0,0 +1,21 @@
+package in.hp.java.userservice.model;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+@Data
+public class User {
```

Comment:
If this Model/DTO class then we should use it for Entity. Please create separate DTO and Entity.

Reply:
Added UserDto